app/fissile_test.go

```diff
@@ -483,6 +502,25 @@ func TestGenerateAuth(t *testing.T) {
 					"apiGroup": "rbac.authorization.k8s.io"
 				}
 			}`,
+			`{
+				"apiVersion": "rbac.authorization.k8s.io/v1",
+				"kind": "ClusterRoleBinding",
+				"metadata": {
+					"name": "default-binding-psp"
+				},
+				"subjects": [
+					{
+						"kind": "ServiceAccount",
+						"name": "default",
+						"namespace": "~"
+					}
+				],
+				"roleRef": {
+					"kind": "ClusterRole",
+					"name": "psp-role-nonprivileged",
+					"apiGroup": "rbac.authorization.k8s.io"
+				}
+			}`,
 		},
```

Comment:
I will check

Reply:
Checked. Found an additional `auth/auth-clusterrole-nonprivileged.yaml`.
Will add that to the test.
Will try to extend the test to detect when there are more files present than expected.

Note: A file for priviledged should not be expected, because all jobs in the input manifest are defaulting to non-privileged, i.e. privileged is not referenced by the manifest, thus not by the chart, and no file is needed for it.